src/internal/views/nunjucks/gauging-stations/licences.njk

```diff
@@ -0,0 +1,223 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "badge.njk" import badge %}
+
+<!--set title-->
+{% set title = [data.stations[data.stationID].riverName] + " at " + [data.stations[data.stationID].label] %}
+
+{% block pageTitle %}
+  {{title}} - GOV.UK
+{% endblock %}
+
+{% block content %}
+  <!-- page title -->
+  <div class="govuk-grid-row govuk-!-margin-bottom-6">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l govuk-!-margin-bottom-1">
+        <span class="govuk-caption-l">{{[data.stations[data.stationID].catchmentName]}}</span>
+        {{title}}
+      </h1>
+      {% set metaData %}
+        {{ govukSummaryList({
+        classes: 'govuk-summary-list--no-border',
```

Comment:
Similar to comment below, if this page renders a single station, why not have 

`station.`

rather than

`data.stations[data.stationId]`

in the view model?